Playbooks/AS-Revoke-Azure-AD-User-Session-From-Incident/Images/init

```diff
@@ -0,0 +1 @@
+
```

Comment:
Hi @AcceleryntSecurityDev, can you please remove this file as it seems to be empty?

Reply:
@v-rbajaj this is done, thanks